detekt_custom.yml

```diff
@@ -747,6 +748,7 @@ datadog:
       # endregion
       # region Kotlin Collections
       - "kotlin.Array.all(kotlin.Function1)"
+      - "kotlin.Array.associateBy(kotlin.Function1)"
```

Comment:
is it used anywhere in this PR?

Reply:
It's used in the TLVBlockType in order to associate the short values with the field names